core/kubernetes/openhim/openhim-mongo-deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: openhim-mongo-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: openhim-mongo
+  template:
+    metadata:
+      labels:
+        component: openhim-mongo
+    spec:
+      volumes:
+        - name: openhim-mongo-storage
+          persistentVolumeClaim:
+            claimName: openhim-mongo-volume-claim
+      containers:
+        - name: openhim-mongo
+          image: mongo
```

Comment:
I think lets supply the version of mongo for this image so that we know which version is installed when looking at the scripts